c2corg_api/models/mailinglist.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.functions import func
+
+from c2corg_api.models import Base, sympa_schema, users_schema
+from c2corg_api.models.user import User
+
+
+class Mailinglist(Base):
+    """
+    Class containing mailing list subscriptions.
+    Based upon Sympa mailing list server https://www.sympa.org/
+    """
+    __tablename__ = 'subscriber_table'
+    __table_args__ = {"schema": sympa_schema}
+
+    listname = Column('list_subscriber', String(50), primary_key=True)
+    email = Column('user_subscriber', String(200), primary_key=True)
+    user_id = Column(
```

Comment:
I would have used `user_id` instead of `email` for the primary key. Because the email depends on the user id.